conf/application.conf

```diff
@@ -129,14 +129,16 @@ microservice {
   }
 }
 
+# These are not needed here as they are defined in the ConnectorHelper now
 mapping {
-  nps {
+  if {
     correlation-id = "correlationid"
     request-type = "gov-uk-originator-id"
     staff-id = "gov-uk-user-id"
   }
 }
 
-httpHeadersWhitelist += ${mapping.nps.correlation-id}
-httpHeadersWhitelist += ${mapping.nps.request-type}
-httpHeadersWhitelist += ${mapping.nps.staff-id}
+# should these not be whitelisting our own request headers that PEGA will send to us (not IF headers)?
```

Comment:
No. We need to whitelist the headers which will be in the HeaderCarrier for the request to IF.
The whitelisting mechanism takes place while generating the HeaderCarrier.